amida-tech/indaba-client
# INBA-683 - No discussion entry when the task is complete.

#### What does this PR do?
When a task isn't complete, you should see the controls to enter a discussion for it.
When a task is complete, you no longer should see those controls.

#### Related JIRA tickets:
https://jira.amida-tech.com/browse/INBA-683

#### How should this be manually tested?
Go to a task that is complete. You should no longer have the ability to enter an entry, mark as resolved, notify a user or cancel/submit a discussion flag.

Then go to a task that isn't complete. The form should reappear. 

#### Background/Context
Another PR is coming that should prevent a task from being complete if there are any lingering discussion flags.

#### Screenshots (if appropriate):
